Dockerfile.try

```diff
@@ -0,0 +1,133 @@
+FROM postgres:15-bullseye
+
+SHELL ["/bin/bash", "-c"]
+
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends \
+        ca-certificates \
+        gnupg \
+        lsb-release \
+        wget && \
+    sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list' && \
+    wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | gpg --dearmor | tee /etc/apt/trusted.gpg.d/apt.postgresql.org.gpg >/dev/null && \
+    apt-get update -y -qq --fix-missing && \
+    apt-get install -y --no-install-recommends \
+        build-essential \
+        clang \
+        clang-11 \
+        gcc \
+        git \
+        jq \
+        libssl-dev \
+        llvm-11 \
+        make \
+        postgresql-server-dev-15 \
+        pkg-config && \
```

Comment:
is there any of this list we can remove?  Like are the `clang*` and `llvm*` packages required?  What about `make` and `jq` and even `gcc`?

Reply:
hmm... it may be that pgx itself needs most of those.  Probably don't need `jq` tho.  Unless it's used here in the Dockerfile?

Reply:
Yeah I believe most of those are needed for the compilation of various things. `jq` is needed within the Dockerfile itself to parse out the exact version of pgx to use (as opposed to hard-coding a version value and grepping it later): https://github.com/tcdi/plrust/pull/287/files#diff-81320fd3745575634e7102a717f7a28356d7a9c647d5bdd4d43dfbbc4e41c61dR46

Reply:
@thomcc any comment regarding anything that could be removed here?

Reply:
Now that I am re-examining it, `TARGET_VERSION` is a really dumb name for that environment variable. Ill rename it to something more meaningful.